sdks/python/apache_beam/runners/interactive/recording_manager.py

```diff
@@ -466,3 +465,25 @@ def record(self, pcolls, max_n, max_duration):
     self._recordings.add(recording)
 
     return recording
+
+  def read(self, pcoll_name: str, pcoll: beam.pvalue.PCollection):
+    # type: (str, beam.pvalue.PValue) -> Union[None, ElementStream]
+
+    """Reads an ElementStream of a computed PCollection.
+
+    Returns None if an error occurs. The caller is responsible of validating if
+    the given pcoll_name and pcoll can identify a watched and computed
+    PCollection without ambiguity in the notebook.
+    """
+
+    try:
+      cache_key = CacheKey.from_pcoll(pcoll_name, pcoll).to_str()
+      return ElementStream(
+          pcoll, pcoll_name, cache_key, float('inf'), float('inf'))
```

Comment:
Please add max_n and max_duration as arguments with defaults as 'inf'

Reply:
Acked.

Reply:
Great, ty!